client/src/components/MultiJurisdictionAudit/_mocks.ts

```diff
@@ -209,6 +209,12 @@ export const jaApiCalls = {
     },
     response: { status: 'ok' },
   },
+  getMapData: {
+    url: 'https://cdn.jsdelivr.net/npm/us-atlas@3/counties-10m.json',
+    response: new Response(
+      'https://cdn.jsdelivr.net/npm/us-atlas@3/counties-10m.json'
+    ),
+  },
```

Comment:
why is `getMapData` part of `jaApiCalls`? isn't it only loaded by audit admins, not jurisdiction admins?